### ISSUE TITLE
Error about  @unimodules/core  while installing the module

### ISSUE THREAD

**Issue by durgaswaroop** [DieTime/react-native-date-picker#9]
I am trying to install the `react-native-date-picker` and as given in instructions, I ran

```
npm install @dietime/react-native-date-picker --save
```
This seems to be automatically installing `expo-linear-gradient`. 
When I run the app, I get the following error:
```
error: Error: Unable to resolve module @unimodules/core from /home/user/projects/GoalsTracker/node_modules/expo-linear-gradient/build/NativeLinearGradient.android.js: @unimodules/core could not be found within the project or in these directories:
  node_modules
```

What should be done here? From what I see, `@unimodules/core` is no longer maintained or needed.

**Comment by durgaswaroop** [comment]
If I try to install ` react-native-unimodules` to resolve the error, I get the following error message:
```
 ERROR  ViewPropTypes will be removed from React Native, along with all other PropTypes. We recommend that you migrate away from PropTypes and switch to a type system like TypeScript. If you need to continue using ViewPropTypes, migrate to the 'deprecated-react-native-prop-types' package.
 ERROR  TypeError: Cannot read property 'includes' of undefined, js engine: hermes
 LOG  Running "main" with {"rootTag":71}
 ERROR  Invariant Violation: "main" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes
```

**Comment by DieTime** [comment]
Thanks for finding the bug, I'll see how to fix it soon.
If you see the strength in yourself to make a PR to fix this bug, I will be very happy about it.

**Comment by durgaswaroop** [comment]
What is the solution here? @DieTime 
I am still fairly new to the ecosystem. If it is something with in my ability to resolve, I can definitely take it.

**Comment by DieTime** [comment]
@durgaswaroop Hi. I fixed this bug in version 1.2.1. This was because I accidentally hardcoded the `expo-linear-gradients` version. Fix commit: https://github.com/DieTime/react-native-date-picker/commit/00d26ffc7ef48308ff8a4c1a38593e15b416f624.
